Adyen/UI/List/ListItem.swift

```diff
@@ -73,6 +76,27 @@ public class ListItem: FormItem {
     public func build(with builder: FormItemViewBuilder) -> AnyFormItemView {
         builder.build(with: self)
     }
+    
+    /// Indicates to the ``ListViewController`` that the specific item is currently loading
```

Comment:
Descriptions of start and stop loading don't seem to match and is a bit confusing.
Also, couldn't see any calls to `stopLoading()`

Reply:
Will update the documentation 👌 

Currently `stopLoading()` is not called - it will be from the AddressLookup PR

Reply:
Updated the documentation, see: 15de5dec6d2b3e2fc88ddb22613159123451a171